lib/generators/boring/pronto/gitlab_ci/install/install_generator.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+require "generators/boring/pronto/base_generator"
+require "boring_generators/generator_helper"
+
+module Boring
+  module Pronto
+    module GitlabCi
+      class InstallGenerator < Boring::Pronto::BaseGenerator
+        desc "Adds pronto gem with various extensions and configure for Gitlab CI"
+        source_root File.expand_path("templates", __dir__)
+
+        class_option :ruby_version, type: :string, aliases: "-rv"
+
+        include BoringGenerators::GeneratorHelper
+
+        def add_configuration
+          @ruby_version = options.ruby_version || app_ruby_version
+
+          if File.exists?(".gitlab-ci.yml")
+            add_configuration_to_existing_file
+            add_lint_stage_to_existing_file
```

Comment:
Why is this needed? Won't `add_configuration_to_existing_file` add lint and ruby verson configuration?

Reply:
They are a bit different.

- add_configuration_to_existing_file adds `pronto` block and configure setups required to run Pronto
- add_lint_stage_to_existing_file is adds "lint" to "stages" so that linting is now enabled in Gitlab CI